src/Akeneo/Tool/Bundle/MessengerBundle/Transport/GooglePubSub/PubSubClientFactory.php

```diff
@@ -26,8 +26,11 @@ public function __construct(string $keyFilePath)
 
     public function createPubSubClient(array $config): PubSubClient
     {
-        return new PubSubClient(array_merge([
-            'keyFilePath' => $this->keyFilePath
-        ], $config));
+        if (!empty($this->keyFilePath)) {
+            return new PubSubClient(array_merge([
+                'keyFilePath' => $this->keyFilePath
+            ], $config));
+        }
```

Comment:
```suggestion
        if (null !== $this->keyFilePath) {
            $config = array_merge([
                'keyFilePath' => $this->keyFilePath
            ], $config));
        }
```

it's a non-blocking suggestion.